### PR TITLE
windows support

### PR DESCRIPTION
**Pull request by MolotovCherry**
Just a couple simple edits. I tested it out and the file paths are correct, and it builds / compiles great.

I'm not exactly a seasoned java developer (I nearly never use it), so please excuse me if the code is not ideal. I will try my best to make any alterations if you would like some changes to it.

I also let my formatter format the `lib.rs` file and implemented 2 suggested clippy lints (but if you don't prefer that, I can remove the changes)

### CODE REVIEW

**Comment by FireMasterK** [comment]
We have a problem with building this on linux:
```
Building "x86_64-pc-windows-msvc" with "cross" for target "x86_64-pc-windows-msvc"
[cross] warning: `cross` does not provide a Docker image for target x86_64-pc-windows-msvc, specify a custom image in `Cross.toml`.
```

Do we want to build `x86_64-pc-windows-gnu` as a target instead?

**Comment by FireMasterK** [comment]
I can confirm that `x86_64-pc-windows-gnu` builds fine on linux, however, I'm not sure if it works on Windows.

**Comment by MolotovCherry** [comment]
On windows, this should require the installation of some mingw toolchains I believe

**Comment by FireMasterK** [comment]
> On windows, this should require the installation of some mingw toolchains I believe

I don't think so. Cross uses docker to manage everything by itself, so if you have docker I think that should be enough.

**Comment by MolotovCherry** [comment]
Yeah, if there's a docker image it should be fine. But I can test specifically. I generally never use docker, so I'm going to have to re-enable it first, I'll report back soon

**Comment by MolotovCherry** [comment]
I ran
`cross build --target=aarch64-unknown-linux-gnu --release`
And as expected, works just fine on windows.

File: reqwest-jni/build.gradle.kts [hunk 1]
```diff
@@ -8,4 +8,5 @@ rust {
 
     targets += target("aarch64-unknown-linux-gnu", "libreqwest.so")
     targets += target("x86_64-unknown-linux-gnu", "libreqwest.so")
+    targets += target("x86_64-pc-windows-msvc", "libreqwest.dll")
```
Comment (FireMasterK) [inline reqwest-jni/build.gradle.kts:11]: This should be updated

Comment (MolotovCherry) [reply reqwest-jni/build.gradle.kts]: Done!

**Comment by MolotovCherry** [comment]
Looks good to me. I got the build working and all of the native targets are there 😃

**Comment by FireMasterK** [comment]
When I'm building on Linux, everything builds fine including the Windows build. The targets are defined in `reqwest-jni/build.gradle.kts`

**Review by FireMasterK — APPROVED** [review]

**Pull request merged**